src/status_im/ui/screens/multiaccounts/key_storage/views.cljs

```diff
@@ -139,25 +139,28 @@
                                            :height 16}]]])
 
 (defn keycard-upsell-banner []
-  [react/touchable-highlight {:on-press #(.openURL ^js react/linking "https://get-keycard.status.im/")}
-   [react/view {:background-color (if (= :dark @colors/theme) "#2C5955" "#DDF8F4")
-                :border-radius 16
-                :margin 16
-                :padding-horizontal 12
-                :padding-vertical 8
-                :flex-direction :row}
-    [react/view
+  [react/view {:background-color (if (= :dark @colors/theme) "#2C5955" "#DDF8F4")
```

Comment:
it would be nice to have a function:
`colors/dark-theme?` or something, so it's clear how we should use it

Reply:
i made only small changes in this PR, we had this UI before, i mean its a valid point, but not introduced in this PR ;)